policy/policy_always_online.c

```diff
@@ -123,11 +120,12 @@ parse_device_mode_ind(DBusMessage *message,
       if (data->flight_mode)
       {
         ILOG_INFO("always online: normal mode");
-        cancel_always_online_timer(data);
+        policy_always_online_cancel_timer(data);
         data->timeout_id =
-            g_timeout_add(2000, always_online_timer_cb, data);
+            g_timeout_add(2000, policy_always_online_make_request_cb, data);
 
-        ILOG_INFO("always online waiting 2s for the normal mode to propagate through the rest of the system, timer id is %d",
+        ILOG_INFO("always online waiting 2s for the normal mode to propagate "
```

Comment:
strings may cross 80 chars boundary to ease grepping in the source code (just a note)